src/components/Layout.tsx

```diff
@@ -129,6 +135,22 @@ const Layout = ({ children }) => {
       <FooterContainer>
         <FooterComponent />
       </FooterContainer>
+      {introModalEnabled && (
+        <IntroModal>
+          <RichText
+            key={123}
+            html={
+              '<iframe ' +
+              'width="100%" ' +
+              'style="aspect-ratio: 16 / 9" ' +
+              'src="https://www.youtube.com/embed/MIdk7wCIdzY?autoplay=1" ' +
+              'title="Intro Video" ' +
+              'allowFullScreen>' +
+              '</iframe>'
+            }
+          />
+        </IntroModal>
+      )}
```

Comment:
@juyrjola is that what you meant by "mock content"?

Reply:
I think the idea was that we'd copy the output of an example block from the Admin UI and pass that down into the Layout component to use as mock data. But this is fine for now, we can switch it out when the backend is ready.